pkg/local_workflows/filter_workflow.go

```diff
@@ -0,0 +1,122 @@
+package localworkflows
+
+import (
+	"encoding/json"
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/snyk/go-application-framework/pkg/configuration"
+	"github.com/snyk/go-application-framework/pkg/local_workflows/content_type"
+	"github.com/snyk/go-application-framework/pkg/local_workflows/local_models"
+	"github.com/snyk/go-application-framework/pkg/workflow"
+	"github.com/spf13/pflag"
+)
+
+const (
+	FilterFindingsWorkflowName = "findings.filter"
+)
+
+var WORKFLOWID_FILTER_FINDINGS = workflow.NewWorkflowIdentifier(FilterFindingsWorkflowName)
+
+func InitFilterFindingsWorkflow(engine workflow.Engine) error {
+	flags := pflag.NewFlagSet(FilterFindingsWorkflowName, pflag.ExitOnError)
+	_, err := engine.Register(WORKFLOWID_FILTER_FINDINGS, workflow.ConfigurationOptionsFromFlagset(flags), filterFindingsEntryPoint)
+
+	return err
+}
+
+type FindingsFilterFunc func(local_models.FindingResource) bool
+
+func contains(slice []string, item string) bool {
+	for _, s := range slice {
+		if s == item {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
suggestion: consider moving to the `utils` package, there's a duplicate method in `presenters_local_findings`

Reply:
I know, I'm late to the game but there is also a [stdlib function](https://pkg.go.dev/slices#Contains) for this. `slices.Contains()`. Reducing our code by using stdlib functions is always a good idea.